helix-term/src/commands.rs

```diff
@@ -192,248 +232,255 @@ impl MappableCommand {
-        delete_selection, "Delete selection",
-        delete_selection_noyank, "Delete selection without yanking",
-        change_selection, "Change selection",
-        change_selection_noyank, "Change selection without yanking",
-        collapse_selection, "Collapse selection into single cursor",
-        flip_selections, "Flip selection cursor and anchor",
-        ensure_selections_forward, "Ensure all selections face forward",
-        insert_mode, "Insert before selection",
-        append_mode, "Append after selection",
-        command_mode, "Enter command mode",
-        file_picker, "Open file picker",
-        file_picker_in_current_directory, "Open file picker at current working directory",
-        code_action, "Perform code action",
-        buffer_picker, "Open buffer picker",
-        jumplist_picker, "Open jumplist picker",
-        symbol_picker, "Open symbol picker",
-        select_references_to_symbol_under_cursor, "Select symbol references",
-        workspace_symbol_picker, "Open workspace symbol picker",
-        diagnostics_picker, "Open diagnostic picker",
-        workspace_diagnostics_picker, "Open workspace diagnostic picker",
-        last_picker, "Open last picker",
-        prepend_to_line, "Insert at start of line",
-        append_to_line, "Append to end of line",
-        open_below, "Open new line below selection",
-        open_above, "Open new line above selection",
-        normal_mode, "Enter normal mode",
-        select_mode, "Enter selection extend mode",
-        exit_select_mode, "Exit selection mode",
-        goto_definition, "Goto definition",
-        add_newline_above, "Add newline above",
-        add_newline_below, "Add newline below",
-        goto_type_definition, "Goto type definition",
-        goto_implementation, "Goto implementation",
-        goto_file_start, "Goto line number <n> else file start",
-        goto_file_end, "Goto file end",
-        goto_file, "Goto files in selection",
-        goto_file_hsplit, "Goto files in selection (hsplit)",
-        goto_file_vsplit, "Goto files in selection (vsplit)",
-        goto_reference, "Goto references",
-        goto_window_top, "Goto window top",
-        goto_window_center, "Goto window center",
-        goto_window_bottom, "Goto window bottom",
-        goto_last_accessed_file, "Goto last accessed file",
-        goto_last_modified_file, "Goto last modified file",
-        goto_last_modification, "Goto last modification",
-        goto_line, "Goto line",
-        goto_last_line, "Goto last line",
-        goto_first_diag, "Goto first diagnostic",
-        goto_last_diag, "Goto last diagnostic",
-        goto_next_diag, "Goto next diagnostic",
-        goto_prev_diag, "Goto previous diagnostic",
-        goto_line_start, "Goto line start",
-        goto_line_end, "Goto line end",
-        goto_next_buffer, "Goto next buffer",
-        goto_previous_buffer, "Goto previous buffer",
+        no_op, "Do nothing", [], None
```

Comment:
Sorting the commands by 1) category (`None`, `Movement`, ...) then alphabetically then aligning the different parts vertically would make it much easier to read this block

Reply:
Sorting it alphabetically would separate many related commands that are currently next to eachother, for example `prepend_to_line` and `append_to_line`.